@attr(tags=['advanced'], required_hardware='false')
def test_deployVM_in_sharedNetwork_as_admin_scope_domain_nosubdomainaccess_subdomainadminuser(self):
    '\n\tValiate that ROOT admin is NOT able to deploy a VM for sub domain admin user in a shared network with scope=domain with no subdomain access\n\t'
    self.apiclient.connection.apiKey = self.default_apikey
    self.apiclient.connection.securityKey = self.default_secretkey
    self.vmdata['name'] = (self.acldata['vmD111']['name'] + '-shared-scope-domain-nosubdomainaccess-root-admin')
    self.vmdata['displayname'] = (self.acldata['vmD111']['displayname'] + '-shared-scope-domain-nosubdomainaccess-root-admin')
    try:
        vm = VirtualMachine.create(self.apiclient, self.vmdata, zoneid=self.zone.id, serviceofferingid=self.service_offering.id, templateid=self.template.id, networkids=self.shared_network_domain_d11.id, accountid=self.account_d111.name, domainid=self.account_d111.domainid)
        self.fail('ROOT admin is able to deploy a VM for sub domain admin user in a shared network with scope=domain with no subdomain access')
    except Exception as e:
        self.debug(('When a admin user from a subdomain deploys a VM in a shared network with scope=domain with no subdomain access %s' % e))
        if (not CloudstackAclException.verifyMsginException(e, CloudstackAclException.NOT_AVAILABLE_IN_DOMAIN)):
            self.fail('Error message validation failed when ROOT admin tries to deploy a VM for sub domain admin user in a shared network with scope=domain with no subdomain access')
